static/app/utils/theme.tsx

```diff
@@ -144,24 +197,18 @@ const lightAliases = {
   /**
    * Primary button colors
    */
-  primaryButtonBorder: '#3d328e',
-  primaryButtonBorderActive: '#352b7b',
-
-  /**
-   * Gradient for sidebar
-   */
-  sidebarGradient:
-    'linear-gradient(294.17deg,#2f1937 35.57%,#452650 92.42%,#452650 92.42%)',
```

Comment:
We now have different sidebar gradients for dark vs light mode. This property was moved down below, to inside `lightTheme` and `darkTheme` ⬇️.